LocalSettings.php

```diff
@@ -363,6 +363,15 @@
 $wgCookieWarningGeoIp2 = true;
 $wgCookieWarningGeoIp2Path = '/var/www/html/piwik/misc/GeoLite2-City.mmdb';
 $wgEnablePartialBlocks = true;
+$wgImportSources = array(
+		array(
```

Comment:
Only one array please

Reply:
Okay, fair enough. I wasn't sure if that was specifically required or not, as I was just copying the code for the titleblacklist sources.